src/microbe_stage/Membrane.tscn

```diff
@@ -22,9 +23,28 @@ shader_parameter/albedoTexture = ExtResource("3")
 shader_parameter/damagedTexture = ExtResource("4")
 shader_parameter/dissolveTexture = ExtResource("5")
 
-[node name="Membrane" type="MeshInstance3D"]
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_v717i"]
+resource_local_to_scene = true
+render_priority = 0
+shader = ExtResource("6_w5050")
+shader_parameter/wigglyNess = 1.0
+shader_parameter/movementWigglyNess = 1.0
+shader_parameter/waviness = 20.0
+shader_parameter/waveSpeed = 15.0
+shader_parameter/waveCutoff = 0.4
+shader_parameter/fade = 0.0
+shader_parameter/tint = Color(0.5, 0.5, 0.95, 1)
+
+[sub_resource type="PlaceholderMesh" id="PlaceholderMesh_gt3ok"]
+
+[node name="Membrane" type="MeshInstance3D" node_paths=PackedStringArray("engulfAnimationMeshInstance")]
 process_priority = 2
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0.00755703, 0, 0.034008)
 mesh = SubResource("1")
 script = ExtResource("1")
-MaterialToEdit = SubResource("2")
+MembraneShaderMaterial = SubResource("ShaderMaterial_dmoyr")
+EngulfShaderMaterial = SubResource("ShaderMaterial_v717i")
+engulfAnimationMeshInstance = NodePath("EngulfMesh")
+
+[node name="EngulfMesh" type="MeshInstance3D" parent="."]
+visible = false
+mesh = SubResource("PlaceholderMesh_gt3ok")
```

Comment:
Is it possible to not set any mesh here? That would slightly simplify this commonly used scene that is instantiated quite a lot.